core/string/ustring.h

```diff
@@ -531,21 +531,13 @@ struct FileNoCaseComparator {
 };
 
 template <typename L, typename R>
-_FORCE_INLINE_ bool is_str_less(const L *l_ptr, const R *r_ptr) {
+_FORCE_INLINE_ int64_t str_compare(const L *l_ptr, const R *r_ptr) {
```

Comment:
This change breaks something somewhere else, I'd suggest adding a new method instead to make sure this doesn't break other code, easier than trying to squash issues where ever this is used

Reply:
it was a case of `l - r` always resulting in a positive integer, oops, fixed